packages/turbo/android/src/main/java/com/reactnativeturbowebview/RNVisitableView.kt

```diff
@@ -122,18 +134,18 @@ class RNVisitableView(context: Context) : LinearLayout(context), SessionSubscrib
     performVisit(restoreWithCachedSnapshot = false, reload = true)
   }
 
-  private fun initializePullToRefresh(turboView: TurboView) {
+  private fun initializePullToReload(turboView: TurboView) {
```

Comment:
Is we rename this, we might as well also rename `isRefreshing` to `isReloading`.

Reply:
Or maybe not. Not sure. What do you think? It's somewhat confusing that the Android uses `setOnRefreshListener` here.

Hm.

I mean; `setOnRefreshListener` refers to the Android UI refreshing action, which _reloads_ our session. We then also have a separate `refresh(..)` method that is totally unrelated (and something else) than `reload()` which refreshes the session's page.

Reply:
I think we should keep the old names, `initializePullToRefresh` etc. and not rename those.